canteens/studierendenwerk.py

```diff
@@ -115,14 +116,12 @@ def parse_menu(menu_html):
             price = item.find('div', class_='text-right').text.strip()
             price_exp = re.compile(r'€ (\d,\d+).*$')
             price = price_exp.sub('*\g<1>€*', price)
-            text = '%s%s %s: %s\n' % (text, annotation, title, price)
-    if text.strip() == '':
-        return ''
-    else:
-        lines = text.split('\n')
-        lines.sort()
-        text = '\n'.join(lines)
-        return '*Speiseplan*%s' % text
+            group_text += '%s %s: %s\n' % (annotation, title, price)
```

Comment:
As a test I added the last line. It sorts all meals of the current group alphabetically and because every line is annotated with an emoji it will sort the group by vegan/veggie/meat/fish. It would also be possible to collect all meals of the group in a list, sort the list and then join them to a string - the following implementation is just a quick example.
What do you think?

```suggestion
            group_text += '%s %s: %s\n' % (annotation, title, price)
        group_text = '\n'.join(sorted(group_text.split('\n'))).strip()
```